netlink_unspecified.go

```diff
@@ -279,11 +279,3 @@ func NeighList(linkIndex, family int) ([]Neigh, error) {
 func NeighDeserialize(m []byte) (*Neigh, error) {
 	return nil, ErrNotImplemented
 }
-
-func SocketGet(local, remote net.Addr) (*Socket, error) {
-	return nil, ErrNotImplemented
-}
-
-func SocketDestroy(local, remote net.Addr) (*Socket, error) {
```

Comment:
Note that the current implementation of `SocketDestroy` in `socket_linux.go` is:
```
func SocketDestroy(local, remote net.Addr) error
```
and the `SocketDestroy` in `netlink_unspecified.go` was therefore incorrect.